bin/generate-values

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+function usage() {
+    echo
+    echo "usage: $(basename $0) <environment> <release-name> [output-file-template]"
+    echo
+}
+
+ROOT=$(realpath $(dirname $(realpath $BASH_SOURCE))/..)
+
+ENVIRONMENT="$1"
+RELEASE="$2"
+
+OUTPUT="${ROOT}/k8s/argocd/${ENVIRONMENT}/${RELEASE}.values.yaml"
+HELMFILE="k8s/helmfile/helmfile.yaml"
```

Comment:
This is now a bit sucky; ideally I wanted to use $ROOT here as well and/or make it possible to specify the path to a helmfile (so the script can be more easily used by CI), but the behaviour of helmfile is making this a bit hard iirc